load_balance.go

```diff
@@ -227,17 +227,21 @@ func connectLoadBalanced(ctx context.Context, config *ConnConfig) (c *Conn, err
 		return connect(ctx, config) // fallback to original behaviour
 	}
 	if lbHost.hostname == config.Host {
+		config.Fallbacks = config.Fallbacks[:1]
```

Comment:
Provide a comment explaining the logic as to why leaving out the first fallback option is required.